packages/superset-ui-chart/test/components/SuperChart.test.tsx

```diff
@@ -62,28 +62,28 @@ describe('SuperChart', () => {
       setTimeout(() => {
         expect(wrapper.render().find('div.test-component')).toHaveLength(1);
         done();
-      }, 5);
+      }, 0);
```

Comment:
We could remove `setTimeout` altogether on all except for slow charts, perhaps? 
```
expect(...);
done();
```

Reply:
tried and it fails tests if I remove `setTimeout`. This might be something to do with `react-loadable`

Reply:
I've found that sometimes `setTimeout(fn, 0)` is needed to get to the next tick for some lifecycle or mount-based functionality.

Reply:
Interesting that the "requeueing" of `fn` is necessary sometimes. We can leave this as-is then.